version.go

```diff
@@ -25,7 +25,7 @@ const (
 // versioning 2.0.0 spec (http://semver.org/).
 const (
 	appMajor uint = 1
-	appMinor uint = 1
+	appMinor uint = 2
```

Comment:
I guess the reason for incrementing the version number is because of the change of config file format. We don't have a `1.1.0` tag yet, should we create one before merging this PR?

Reply:
Good catch, got ahead of myself there. Will revert.